src/core/moves/SliceSamplingMove.cpp

```diff
@@ -14,8 +17,14 @@
 #include "RbOrderedSet.h"
 #include "StochasticNode.h"
 
+using boost::optional;
+
 using namespace RevBayesCore;
 
+const double log_0 = RbConstants::Double::min;
+
+// How can we make a run-time adjustable log level?
+int log_level = 0;
```

Comment:
should the log level be a static member ?

Reply:
Actually I was thinking that revbayes should have an option for a logging level, and this is a placeholder for it.  Right now revbayes has options for `linewidth`, 'outputPrecision', and a few other things (see `core/utils/RbSettings.cpp`).

Reply:
I think it would fine to add the log level directly to RbSettings 
and we can make a note that this exists so more people can use it in their own code - maybe file it as a "feature request" in the issues ?